cclib/parser/psi4parser.py

```diff
@@ -102,6 +102,13 @@ def extract(self, inputfile, line):
             if self.reference[0] == 'C':
                 self.reference = self.reference[1:]
 
+        # Parse the XC density functional

```

Comment:
Are you using `metadata` for anything? If so, would be willing to offer us input on what you'd like it to contain? We have multiple running issues: https://github.com/cclib/cclib/issues/451, https://github.com/cclib/cclib/issues/523, https://github.com/cclib/cclib/pull/599.